prog/mutation.go

```diff
@@ -258,15 +258,18 @@ func mutateInt(r *randGen, s *state, arg Arg) (calls []*Call, retry, preserve bo
 	if r.bin() {
 		return regenerate(r, s, arg)
 	}
+	argSize := uint64(math.Min(float64(arg.Size()), 8))
 	a := arg.(*ConstArg)
 	switch {
 	case r.nOutOf(1, 3):
 		a.Val += uint64(r.Intn(4)) + 1
 	case r.nOutOf(1, 2):
 		a.Val -= uint64(r.Intn(4)) + 1
 	default:
-		a.Val ^= 1 << uint64(r.Intn(64))
+		bits := int(8 * argSize)
+		a.Val ^= 1 << uint64(r.Intn(bits))
```

Comment:
This is good - now we are flipping only bits that will actually go into the kernel.